### PR TITLE
feat: increment/decrement example

### PR DESCRIPTION
**Pull request by liemgumball**
- Simple increment/decrement example
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 node_modules
 .parcel-cache
-dist
\ No newline at end of file
+dist
+package-lock.json
```
Comment (huyenpham-agilityio) [inline .gitignore]: why ignore this?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+.prettierrc
```
Comment (huyenpham-agilityio) [inline .gitignore]: ditto

File: react/hello-world/package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "hello-world",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```
Comment (huyenpham-agilityio) [inline react/hello-world/package.json:5]: double check what is dependencies/devDependencies

File: react/hello-world/src/App.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
+.App {
```
Comment (huyenpham-agilityio) [inline react/hello-world/src/App.css]: naming, check other cases please

File: react/hello-world/src/components/header/index.jsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Link, useLocation } from 'react-router-dom'
+import './Header.scss'
+
+const Header = () => {
+  const location = useLocation()
+  return (
```
Comment (huyenpham-agilityio) [inline react/hello-world/src/components/header/index.jsx]: break line before return

File: react/hello-world/src/components/pages/home/index.jsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
+import './Home.scss'
+import { useState } from 'react'
+import { FaPlus, FaMinus } from 'react-icons/fa'
+import Button from '../../button'
+
+const Home = () => {
+  const [counter, setCounter] = useState(0)
```
Comment (huyenpham-agilityio) [inline react/hello-world/src/components/pages/home/index.jsx]: define type

Comment (liemgumball) [reply react/hello-world/src/components/pages/home/index.jsx]: did you mean that I should write it in TypeScript?

File: react/hello-world/src/App.css (hunk 4 above)
Comment (liemgumball) [reply react/hello-world/src/App.css]: this is the default App.css given by React

Comment (huyenpham-agilityio) [reply react/hello-world/src/App.css]: you can update it

File: react/hello-world/src/components/pages/home/index.jsx (hunk 6 above)
Comment (huyenpham-agilityio) [reply react/hello-world/src/components/pages/home/index.jsx]: yes, you learn TS before, so please apply it

**Pull request merged**